src/Corrade/Utility/AndroidLogStreamBuffer.cpp

```diff
@@ -33,7 +34,10 @@ AndroidLogStreamBuffer::~AndroidLogStreamBuffer() = default;
 
 int AndroidLogStreamBuffer::sync() {
     /* Write the data */
-    __android_log_write(std::int32_t(_priority), _tag.data(), str().data());
+    const char * tag = _tag.data();
+    const char * data = str().data();
+    __android_log_write(std::int32_t(_priority), tag, data);
+    std::cout << data;
```

Comment:
```suggestion
    const std::string data = str();
    __android_log_write(std::int32_t(_priority), _tag.data(), data.data());
    std::cout << data;
```

With the original version, you'd pass a null-terminated array to cout, which means it'd stop on the first null byte (and thus when something goes wrong *real bad*, the message will be cut off). Since there already is a string with size information, just use it.